src/Filter/Filter.php

```diff
@@ -5,13 +5,13 @@
 namespace Paraunit\Filter;
 
 use Paraunit\Configuration\PHPUnitConfig;
-use Paraunit\Proxy\PHPUnitUtilXMLProxy;
+use PHPUnit\Util\Xml\Loader;
```

Comment:
Heh, relying on `@internal` classes is mandatory indeed, from now on, if we don't want to reinvent the wheel :+1:

Reply:
Yup, I noticed... If it breaks, I'll decide if I want to copy that over or not.

Reply:
I'm confident that every sudden change introduced by Sebastian will always be for a more comprehensive and stable solution for the future, so keeping up to phpunit changes are going to be easier then rewriting it, I hope.